polls/models.py

```diff
@@ -12,13 +11,27 @@
 class Question(Page):
     subpage_types = ['polls.Choice']
 
+    def can_vote(user):
```

Comment:
you're missing `self` as the first argument of a class method, it should be `def can_vote(self, user)`